gwpy/cli/qtransform.py

```diff
@@ -149,6 +151,11 @@ def fformat(x):  # float format
                 return '[{0}]'.format(', '.join(map(fformat, x)))
             if isinstance(x, Quantity):
                 x = x.value
+            elif isinstance(x, str):
+                self.log(0, 'WARNING: fformat called with a string. This has '
+                            'been depricated and may disappear '
+                            'in a future release.')
```

Comment:
@areeda, can we try using `warnings.warn(..., DeprecationWarning)` instead, so that the warnings can be caught easily during testing?

Reply:
warnings print the line that caused the warning to be raised, which here would just be the warning itself. I think that's a small price to pay for the ability to promote this warning to an error during testing to find if any code actually will touch this clause.

Reply:
or, we could just do something like this:

```python
        if isinstance(x, Quantity):
            x = x.value
        return '{0:.2f}'.format(float(x))
```

and cast everything to float regardless. This would only then error when non-`float()`able values present, which is an bug anyway.